File: benchmarkDS/ourMethod/OPENAI_gpt-5-2025-08-07/Lse27.py
from typing import Any


def compute_total_sales(month1: int, month2: int, month3: int) -> int:
    if not all(isinstance(v, int) for v in (month1, month2, month3)):
        raise ValueError("invalid input")
    return month1 + month2 + month3


def main() -> None:
    totalSales = 0
    totalSales += 100
    totalSales += 200
    totalSales += 300
    print(f"Total Sales: {totalSales}")

    totalSales = 0
    totalSales += 0
    totalSales += 0
    totalSales += 0
    print(f"Total Sales: {totalSales}")

    totalSales = 0
    totalSales += 999
    totalSales += 1
    totalSales += 0
    print(f"Total Sales: {totalSales}")

    totalSales = 0
    totalSales += 12345
    totalSales += 67890
    totalSales += 111
    print(f"Total Sales: {totalSales}")

    totalSales = 0
    totalSales += -50
    totalSales += 100
    totalSales += 200
    print(f"Total Sales: {totalSales}")


if __name__ == "__main__":
    main()

File: benchmarkDS/Vanilla/OPENAI_gpt-5-2025-08-07/Lse27.c
#include <stdio.h>

double computeTotalSales(double m1, double m2, double m3) {
    return m1 + m2 + m3;
}

int main(void) {
    double totalSales = 0;
    totalSales += 120.50;
    totalSales += 230.75;
    totalSales += 349.99;
    printf("%f\n", totalSales);

    totalSales = 0;
    totalSales += 0.0;
    totalSales += 0.0;
    totalSales += 0.0;
    printf("%f\n", totalSales);

    totalSales = 0;
    totalSales += 1000.00;
    totalSales += 500.25;
    totalSales += 250.25;
    printf("%f\n", totalSales);

    totalSales = 0;
    totalSales += 10.10;
    totalSales += 20.20;
    totalSales += 30.30;
    printf("%f\n", totalSales);

    totalSales = 0;
    totalSales += 99999.99;
    totalSales += 1.01;
    totalSales += 0.00;
    printf("%f\n", totalSales);

    return 0;
}